### PR TITLE
Add 'mtu' entry for libvirt network

### PR DESCRIPTION
**Pull request by chloerh**
'mtu' has only 1 attrib: size.

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

File: virttest/libvirt_xml/network_xml.py [hunk 1]
```diff
@@ -371,7 +371,7 @@ class NetworkXMLBase(base.LibvirtXMLBase):
                  'autostart', 'persistent', 'forward', 'mac', 'ip',
                  'bandwidth_inbound', 'bandwidth_outbound', 'portgroup',
                  'dns', 'domain_name', 'nat_port', 'forward_interface',
-                 'routes', 'virtualport_type')
+                 'routes', 'virtualport_type', 'mtu')
```
Comment (balamuruhans) [inline virttest/libvirt_xml/network_xml.py]: I believe `mtu` entry have to be added in `class Interface('network')` in virttest/libvirt_xml/devices/interface.py

https://libvirt.org/formatdomain.html#mtu

Comment (chloerh) [reply virttest/libvirt_xml/network_xml.py]: Yes, and mtu entry also exists in network.

**Comment by balamuruhans** [comment]
@chloerh please rebase this PR.

**Comment by chloerh** [comment]
@balamuruhans updated, thanks

**Comment by kylazhang** [comment]
@chloerh please merge the 2 commit to 1, thanks

**Comment by balamuruhans** [comment]
Thanks @kylazhang @chloerh, merging this PR.

**Pull request merged**